website/sidebars.js

```diff
@@ -12,8 +12,13 @@ module.exports = {
       "rbe-github-actions",
       "rbe-pools",
     ],
-    "Workflows (CI)": ["workflows-introduction", "workflows-setup", "workflows-config"],
-    "Remote Bazel": ["remote-bazel-introduction"],
+    "Remote Runners (CI)": [
+      "remote-runner-introduction",
+      "remote-runner-features",
+      "workflows-setup",
+      "workflows-config",
+      "remote-bazel-introduction",
```

Comment:
should we rename this one to just "remote bazel"? since we sort of introduced it in the remote-runner-introduction article